### PR TITLE
install py36-sqlite during make install-dev on FreeBSD

### PR DESCRIPTION
**Pull request by gronke**
Addresses a dependency issue of MyPy, that began to depend on sqlite3.

see https://github.com/python/mypy/issues/6207

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ deps:
 install: deps
 	python3.6 -m pip install -U .
 install-dev: deps
-	if [ "`uname`" = "FreeBSD" ]; then pkg install -y gmake; fi
+	if [[ "`uname`" = "FreeBSD" ]]; then pkg install -y gmake py36-sqlite3; fi
```
Comment (igalic) [inline Makefile]: you sure this change works on BSD's sh?

**Comment by gronke** [comment]
sqlite3 needs to be installed on Linux as well.

**Comment by igalic** [comment]
i meant the [] vs [[]] syntax change

**Comment by gronke** [comment]
> i meant the [] vs [[]] syntax change

Yes, that works. Run `make install-dev` and see it yourself 😄

**Comment by igalic** [comment]
```
E: Could not open lock file /var/lib/dpkg/lock - open (13: Permission denied)
E: Unable to lock the administration directory (/var/lib/dpkg/), are you root?
make: *** [install-travis] Error 100
The command "make install-travis" failed and exited with 2 during .
```

we should just add extra packages to our .travis.yml https://docs.travis-ci.com/user/installing-dependencies/#adding-apt-packages

File: Makefile [hunk 2]
```diff
@@ -11,7 +11,8 @@ deps:
...
+	apt install -y sqlite3 python3 2>/dev/null || true
 	python3.6 -m pip install -Ur requirements-dev.txt
```
[the remark this answers is not in the document]
Comment (gronke) [reply Makefile]: Can you please elaborate what this means?

[the remark this answers is not in the document]
Comment (igalic) [reply Makefile]: you're apt installing sqlite3 here, even tho we already added it to travis.yml
The makefile is a BSD makefile.

**Pull request merged**